app/lib/server/startup/settings.js

```diff
@@ -354,7 +354,7 @@ settings.addGroup('Accounts', function() {
 			public: true,
 			i18nLabel: 'Group_by_Type',
 		});
-		this.add('Accounts_Default_User_Preferences_sidebarViewMode', 'medium', {
+		this.add('Accounts_Default_User_Preferences_sidebarViewMode', 'extended', {
```

Comment:
This is not necessary, we can leave it as it was.

Reply:
If we are setting extended as default, then in the admin options it is used to reset to default mode.